.github/workflows/tests.yml

```diff
@@ -0,0 +1,45 @@
+name: tests
+on: 
+  workflow_dispatch:
+  push:
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+        BATON_PUBLIC_KEY: ${{ secrets.BATON_PUBLIC_KEY }}
+        BATON_PRIVATE_KEY: ${{ secrets.BATON_PRIVATE_KEY }}
+        BATON_LOG_LEVEL: 'debug'
+        # Revoke grants variable
+        REVOKE_GRANT: 'team:65de391acee49637479ff761:member:user:6000932cdb25974d816dd711'
+        # Grant entitlements variables
+        GRANT_ENTITLEMENT: 'team:65de391acee49637479ff761:member'
+        GRANT_PRINCIPAL: '6000932cdb25974d816dd711'
+        GRANT_PRINCIPAL_TYPE: 'user'
```

Comment:
Thanks for making them environment variables. I'll change these once I change the github secrets.

Reply:
ok